### PR TITLE
Added Python API links to the Docs

### PR DESCRIPTION
**Pull request by jasondaming**

### CODE REVIEW

**Review by sciencewhiz — COMMENTED** [review]
@virtuald said the python API docs location will change. https://github.com/wpilibsuite/.github/pull/2#issue-2011105271

**Comment by jasondaming** [comment]
I assume I can update this to use intersphinx once https://github.com/wpilibsuite/frc-docs/pull/2417 is merged.

@virtuald will the same style intersphinx link for replacing these urls?  These links will persist through any location changes you plan to make right?

**Comment by virtuald** [comment]
This one could also be intersphinx, but I think you'd want to link to a documentation header, not a particular class or module.

Let me try to get my ducks in a row tonight.

**Comment by jasondaming** [comment]
Do we need to do this a different way or is this sufficient?

File: source/index.rst [hunk 1]
```diff
@@ -362,6 +371,7 @@ Community translations can be found in a variety of languages in the bottom-left
 
    WPILib Java API Docs <https://github.wpilib.org/allwpilib/docs/beta/java/index.html>
    WPILib C++ API Docs <https://github.wpilib.org/allwpilib/docs/beta/cpp/index.html>
+   WPILib Python API Docs <https://robotpy.readthedocs.io/projects/wpilib/en/stable/api.html>
```
Comment (virtuald) [inline source/index.rst:374]: URL needs to be https://robotpy.readthedocs.io/projects/robotpy/en/latest/

File: source/index.rst [hunk 2]
```diff
@@ -229,6 +229,15 @@ Community translations can be found in a variety of languages in the bottom-left
 
          C++
 
+      .. button-link:: https://robotpy.readthedocs.io/projects/wpilib/en/stable/api.html
```
Comment (sciencewhiz) [inline source/index.rst:232]: This should also be updated. https://robotpy.readthedocs.io/projects/robotpy/en/latest/

**Review by sciencewhiz — APPROVED** [review]

**Pull request merged**